app/database.server.ts

```diff
@@ -0,0 +1,53 @@
+import { db } from "./firebase.server";
```

Comment:
This file contains all the database operations:
- create a user profile
- update a user profile
- get a user profile by slug (aka GitHub login)